Dockerfile

```
@@ -0,0 +1,9 @@
+FROM python:3.12-alpine
+
+RUN mkdir /app
+COPY src /app/src
+COPY requirements/prod.txt ./requirements-prod.txt
+RUN python -m pip install -r ./requirements-prod.txt
```

nit: If we are copying, we could always call it requirements.txt so the following line looks more standard

```suggestion
COPY requirements/prod.txt ./requirements.txt
RUN python -m pip install -r ./requirements.txt
```

Also do we need the `python -m` here (since we pulling a python image)?

---

Strictly speaking `python -m pip` vs `pip` doesn't make a difference inside the container since it's a pre-baked python environment anyway. It's just that invoking `pip` as a module is muscle memory for me at this point :)